# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/xslt/tests/XSLTMark/XSLTMark-static.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c dom/xslt/tests/XSLTMark/XSLTMark-static.js
--- a/dom/xslt/tests/XSLTMark/XSLTMark-static.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/dom/xslt/tests/XSLTMark/XSLTMark-static.js	Tue Feb 12 16:08:25 2019 +0000
@@ -30,7 +30,7 @@
     });
     var instream = 
         Cc[SIS_CTRID].createInstance(nsISIS);
-    instream.init(chan.open2());
+    instream.init(chan.open());
 
     return instream.read(instream.available());
 }